R/zzz.R

```diff
@@ -82,9 +82,10 @@ is_S7_type <- function(x) {
 }
 
 check_subsettable <- function(x, allow_env = FALSE) {
-  allowed_types <- c("list", if (allow_env) "environment")
+  allowed_types <- c("list", if (allow_env) "environment",
+                     "language", "pairlist")
   if (!typeof(x) %in% allowed_types) {
-    stop("S7 objects are not subsettable.", call. = FALSE)
+    stop("S7 objects are not subsettable.", call. = TRUE)
```

Comment:
We don't usually show the call when it's a non-exported function.

Reply:
We really need a better error call and traceback from custom getter/setter calls, because right now, debugging any errors signaled from them is unnecessarily difficult. But that should be a separate issue.